docs/contributor/performance-tests/metrics/metrics.md

```diff
@@ -0,0 +1,690 @@
+
+Metric count was increased in 1000 steps to find the limit of the metric agent.
+
+Following graph shows max stable load from single target with huge amount of metric data.
+![Peak accepted metric points](./assets/single-target-peak-metric-point.jpg)
+
+Metric agent reached max `7K metric points/sec` from a single metric target, with 6000 and 20 metric series. In this test scenario the 
+ measured raw metric payload size was `~51MB`
+
+Another test was performed to determine the impact of different metric series on metric agent. For this the test metric series were increased from `20` to `30`, distinct metric count started from `1000` and increased in `1000` steps until the limit of metric agent was reached.
+Metric agent reached a peak of `~7K metric points/sec` (yellow line from graph above) with `4000` distinct metric. In this setup the measured raw metric payload size was `~31MB`
+
+In both the configurations metric agent reached a peak of `~7K metric points/sec` independent size of raw metric data, thus raw metric payload size has no impact on metric agent performance.
+
+In both configurations Avalanche load generator generated `120K metric points/scrape` 
+
+> **NOTE:** Avalanche load generator resources have to be changed for this scenario, for this test CPU settings changed to 400m and Memory to 1Gi 
+### Have multiple workloads across different nodes
+
+Setup:
+
+Gardener GCP cluster:
+- Cluster with 15 Nodes
+- Machine types n2-standard-16 (16 CPU, 64Gi memory), Gardener supports max 100 pods per node
+
+Avalanche load generator configuration:
+- Metric Count: 100
+- Metric Series: 20
+- Number of label: 10
+
+Prometheus:
+- Instance memory increased to the 7Gi
+
+To simulate a smooth ramp-up and avoid huge data flood at the beginning, load generator started with `100` instances, instance counts was increased every by `100` instances every 5 minutes until max pod count of cluster reached.
+
+Following graph show metric agent received metrics 
+![Metric agent accepted metrics](./assets/max-pod-agent-accepted-metrics.jpg)
+
+Following graph show metric gateway received metrics
+![Metric gateway accepted metrics](./assets/max-pod-gateway-accepted-metrics.jpg)
+
+Following graph show metric gateway exported metrics
+![Metric gateway exported metrics](./assets/max-pod-gateway-sent-metrics.jpg)
+
+Following graph show metric agent CPU utilization
+![Metric agent exported metrics](./assets/max-pod-agent-cpu.jpg)
+
+Following graph show metric agent memory utilization
+![Metric agent exported metrics](./assets/max-pod-agent-memory.jpg)
+
+Following graph show Kubernetes API server metrics
+![Metric agent exported metrics](./assets/max-pod-k8s-api.jpg)
+
+Test reached max pod count `1430` on cluster, all metrics are successfully received and exported to the metric gateway, no critical memory, cpu or network utilization observed.
+
+Kubernetes API request duration reach for `GET` operations at peak `~750ms` and for `LIST` operations `~900ms`.
```

Comment:
```suggestion
The Kubernetes API request duration reach for `GET` operations at peak was `~750ms`, and for `LIST` operations `~900ms`.
```

Same here, why are some elements formatted in code font? Does any of our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-code-font-for-these-items) apply?